command/server.go

```diff
@@ -2498,6 +2498,15 @@ func (c *ServerCommand) setupTelemetry(config *server.Config) (*metricsutil.Metr
 		return nil, err
 	}
 
+	// Intialize a wrapper around the global sing.
+	wrapper := &metricsutil.ClusterMetricSink{
+		ClusterName:         config.ClusterName,
+		MaxGaugeCardinality: 500,
+		GaugeInterval:       10 * time.Minute,
+		Sink:                fanout,
+	}
+	metricsutil.SetGlobalSink(wrapper)
```

Comment:
I'm a little worried about having this enabled by default. For systems that do not support labels (e.g. statsd) i think this would be a breaking change due to the way that go-metrics flattens the keys. 

https://github.com/armon/go-metrics/blob/38377b616bac08306ad0903d5fb436a5330d2973/statsd.go#L99-L104

Reply:
For now this will only be used for newly-introduced metrics, as stated in the RFC.  The new calls will look like

metricsutil.IncrCounterWithLabels( ... )

instead of 

metrics.IncrCounterWithLabels( ... )

Reply:
Ah okay, that makes sense